crowbar_framework/vendor/assets/stylesheets/fontawesome/_extras.scss

```diff
@@ -0,0 +1,44 @@
+/* EXTRAS
+ * -------------------------- */
+
+/* Stacked and layered icon */
+
+/* Animated rotating icon */
+.#{$fa-css-prefix}-spin {
+  -webkit-animation: spin 2s infinite linear;
```

Comment:
Properties should be ordered -moz-animation, -o-animation, -webkit-animation, animation<br>Avoid vendor prefixes.